idd/Energy+.idd.in

```diff
@@ -4030,442 +4030,506 @@ Schedule:Compact,
- A4143,A4144,A4145,A4146,A4147,A4148,A4149,A4150,A4151,A4152,A4153,A4154,A4155,A4156,A4157, \note fields as indicated
- A4158,A4159,A4160,A4161,A4162,A4163,A4164,A4165,A4166,A4167,A4168,A4169,A4170,A4171,A4172, \note fields as indicated
- A4173,A4174,A4175,A4176,A4177,A4178,A4179,A4180,A4181,A4182,A4183,A4184,A4185,A4186,A4187, \note fields as indicated
- A4188,A4189,A4190,A4191,A4192,A4193,A4194,A4195,A4196,A4197,A4198,A4199,A4200,A4201,A4202, \note fields as indicated
- A4203,A4204,A4205,A4206,A4207,A4208,A4209,A4210,A4211,A4212,A4213,A4214,A4215,A4216,A4217, \note fields as indicated
- A4218,A4219,A4220,A4221,A4222,A4223,A4224,A4225,A4226,A4227,A4228,A4229,A4230,A4231,A4232, \note fields as indicated
- A4233,A4234,A4235,A4236,A4237,A4238,A4239,A4240,A4241,A4242,A4243,A4244,A4245,A4246,A4247, \note fields as indicated
- A4248,A4249,A4250,A4251,A4252,A4253,A4254,A4255,A4256,A4257,A4258,A4259,A4260,A4261,A4262, \note fields as indicated
- A4263,A4264,A4265,A4266,A4267,A4268,A4269,A4270,A4271,A4272,A4273,A4274,A4275,A4276,A4277, \note fields as indicated
- A4278,A4279,A4280,A4281,A4282,A4283,A4284,A4285,A4286,A4287,A4288,A4289,A4290,A4291,A4292, \note fields as indicated
- A4293,A4294,A4295,A4296,A4297,A4298,A4299,A4300,A4301,A4302,A4303,A4304,A4305,A4306,A4307, \note fields as indicated
- A4308,A4309,A4310,A4311,A4312,A4313,A4314,A4315,A4316,A4317,A4318,A4319,A4320,A4321,A4322, \note fields as indicated
- A4323,A4324,A4325,A4326,A4327,A4328,A4329,A4330,A4331,A4332,A4333,A4334,A4335,A4336,A4337, \note fields as indicated
- A4338,A4339,A4340,A4341,A4342,A4343,A4344,A4345,A4346,A4347,A4348,A4349,A4350,A4351,A4352, \note fields as indicated
- A4353,A4354,A4355,A4356,A4357,A4358,A4359,A4360,A4361,A4362,A4363,A4364,A4365,A4366,A4367, \note fields as indicated
- A4368,A4369,A4370,A4371,A4372,A4373,A4374,A4375,A4376,A4377,A4378,A4379,A4380,A4381,A4382, \note fields as indicated
- A4383,A4384,A4385,A4386,A4387,A4388,A4389,A4390,A4391,A4392,A4393,A4394,A4395,A4396,A4397, \note fields as indicated
- A4398,A4399,A4400,A4401,A4402,A4403,A4404,A4405,A4406,A4407,A4408,A4409,A4410,A4411,A4412, \note fields as indicated
- A4413,A4414,A4415,A4416,A4417,A4418,A4419,A4420,A4421,A4422,A4423,A4424,A4425,A4426,A4427, \note fields as indicated
- A4428,A4429,A4430,A4431,A4432,A4433,A4434,A4435,A4436,A4437,A4438,A4439,A4440,A4441,A4442, \note fields as indicated
- A4443,A4444,A4445,A4446,A4447,A4448,A4449,A4450,A4451,A4452,A4453,A4454,A4455,A4456,A4457, \note fields as indicated
- A4458,A4459,A4460,A4461,A4462,A4463,A4464,A4465,A4466,A4467,A4468,A4469,A4470,A4471,A4472, \note fields as indicated
- A4473,A4474,A4475,A4476,A4477,A4478,A4479,A4480,A4481,A4482,A4483,A4484,A4485,A4486,A4487, \note fields as indicated
- A4488,A4489,A4490,A4491,A4492,A4493,A4494,A4495,A4496,A4497,A4498,A4499,A4500; \note fields as indicated
+  A7,A8,A9,A10,A11,A12,A13,A14,A15,A16,A17,A18,A19,A20, \note fields as indicated
```

Comment:
Fields 7-152 are probably quite often used, and as such it seems like you wouldn't want to remove the field names from them.  Is there a reason you needed to do that?

Reply:
No, that was inadvertent. I'll revert those field name changes, thanks for pointing it out.